torch/testing/_internal/common_fsdp.py

```diff
@@ -1118,7 +1150,7 @@ def setUp(self):
 
     @property
     def world_size(self):
-        return min(torch.cuda.device_count(), 8) if torch.cuda.is_available() else 4
+        return min(DEVICE_COUNT, 8)
```

Comment:
I propose `return DEVICE_COUNT` if there is no specific reason to cap at 8. (I know it is not from this PR, appreciate it if you could give it a try).